rust/noosphere-gateway/src/worker/cleanup.rs

```diff
@@ -0,0 +1,297 @@
+use std::time::Duration;
+
+use anyhow::{anyhow, Result};
+use noosphere_core::{
+    context::{HasMutableSphereContext, HasSphereContext, SphereCursor, COUNTERPART},
+    data::Did,
+};
+use noosphere_storage::{KeyValueStore, Storage};
+use strum_macros::Display as EnumDisplay;
+use tokio::{
+    sync::mpsc::{unbounded_channel, UnboundedReceiver, UnboundedSender},
+    task::JoinHandle,
+};
+use tokio_stream::StreamExt;
+
+// Once an hour
+const PERIODIC_CLEANUP_INTERVAL_SECONDS: u64 = 60 * 60;
+
+#[derive(EnumDisplay)]
```

Comment:
TIL

Reply:
This is just an aliased `strum` thing; the exported name is `Display` and it conflicts with `std::fmt::Display` so I usually rename it.